### PR TITLE
Release/r122

### PR DESCRIPTION
**Pull request by istreeter**

### CODE REVIEW

**Review by chuwy — COMMENTED** [review]
Looks good. Just few questions for @AlexBenny.

File: schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.snowplowanalytics.mobile",
+    "name" : "remote_config",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "title": "Snowplow Mobile Trackers config file",
```
[the remark this answers is not in the document]
Comment (AlexBenny) [reply schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0:9]: I initially copied by a schema online so I guess the title property was there and I've copied the format.

[the remark this answers is not in the document]
Comment (chuwy) [reply schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0:9]: Don't think it's worth keeping it then. It just duplicates `description` and inconsistent with other schemas on IC.

File: schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0 [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+      "pattern": "^[0-9]+\\.[0-9]+\\.[0-9a-zA-Z.+\\-]+$"
+    },
+    "configurationVersion": {
+      "description": "Version number that identifier the current configuration. It has to be increased on each update.",
+      "type": "integer"
+    },
+    "configurationBundle": {
+      "description": "The list of configurations for each tracker to configure.",
+      "type": "array",
+      "items": [
```
[the remark this answers is not in the document]
Comment (AlexBenny) [reply schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0:25]: Thanks, I correct it as "items": { ... } as suggested.

File: schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0 [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  "self" : {
+    "vendor" : "com.snowplowanalytics.mobile",
+    "name" : "remote_config",
+    "version" : "1-0-0",
+    "format" : "jsonschema"
+  },
+  "description": "The configuration file for the Snowplow mobile trackers.",
+  "type": "object",
+  "properties": {
+    "configurationVersion": {
```
Comment (paulboocock) [inline schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0:12]: Do we need a configurationVersion? Doesn't the schema version give us that information already?

**Review by paulboocock — APPROVED** [review]
LGTM. Just one comment for @AlexBenny although likely not a blocker as it could just stay as `1` forever if the schema version can be used instead.

**Review by paulboocock — APPROVED** [review]

File: schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0 (hunk 3 above)
Comment (paulboocock) [reply schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-0]: Lets move this discussion back to https://github.com/snowplow/iglu-central/issues/1109 to unblock this release.

**Review by chuwy — APPROVED** [review]

**Pull request merged**